w3c/wai-aria-practices
# Add support for `data-skipto` parameters

See #259. More specifically https://github.com/w3c/wai-aria-practices/issues/259#issuecomment-1730048296 which motivated these changes.

This will require https://github.com/w3c/aria-practices/pull/2807 to be merged first. Also `content/patterns/patterns.html` and `content/patterns/practices.html` will have to be updated with the new skipto script as well in aria-practices.

**Testing Instructions**

**Note:** I created a testing branch on [aria-practices, update-skipto-merged](https://github.com/w3c/aria-practices/tree/update-skipto-merged) which merges 2807 and aria-practices' main, so there aren't any assumptions that have been missed following https://github.com/w3c/aria-practices/issues/2702 being completed and merged which also plays a part in this work.

1. Update `.gitmodules` aria-practices repo branch to `update-skipto-merged`, like the following:

```yml
[submodule "_external/aria-practices"]
  path = _external/aria-practices
  url = https://github.com/w3c/aria-practices.git
  branch = update-skipto-merged
...
```

2. Run `git submodule --remote --recursive` to use the latest sample branch's commit instead.
3. Follow the README instructions from `Install Jekyll` onwards to run project as expected.
4. Observe that the updated `skipto` script and new parameters are being used.
